resources/functionMap_php80delta.php

```diff
@@ -146,6 +146,7 @@
 	'old' => [
 
 		'array_combine' => ['associative-array|false', 'keys'=>'string[]|int[]', 'values'=>'array'],
+		'array_fill' => ['array', 'start_key'=>'int', 'num'=>'0|positive', 'val'=>'mixed'],
```

Comment:
If you add something to the `old` section, you just remove the key from the final map.

Reply:
I am not sure what todo. I moved this line to the `new` section now.

is there anything I can do to test the change to this file?

Reply:
It's not necessary to test it. You can test it yourself simply by analysing file with wrong `array_fill()` call.

Reply:
ok - tested the latest change locally and it works:

```
$ php bin/phpstan analyze test.php --level max
Note: Using configuration file C:\xampp7.3\htdocs\phpstan-src\phpstan.neon.dist.
 1/1 [============================] 100%

 ------ --------------------------------------------------------------------
  Line   test.php
 ------ --------------------------------------------------------------------
  4      Parameter #2 $count of function array_fill expects int<0, max>, -5
         given.
 ------ --------------------------------------------------------------------


 [ERROR] Found 1 error

```